src/typeInference/annotateDynamicFunctionInvocationExpr.ts

```diff
@@ -0,0 +1,22 @@
+import { SequenceType } from '../expressions/dataTypes/Value';
+import StaticContext from '../expressions/StaticContext';
+import { IAST } from '../parsing/astHelper';
+
+/**
+ * At this moment there is no way to infre the return type of this function as
```

Comment:
Infre!

Reply:
Well yes, we are doing type infrerence here.

Reply:
gooey said we did